Update CHANGELOG for next bundle release (#96)
giantswarm/security-bundle: CHANGELOG.md

### Changed

- Changed the value type of configmap/secret values to object rather than multi line string.

> - Update to `kyverno` (app) version 0.14.4, setting a default namespace for `PolicyExceptions`.

Reviewer: ```suggestion
- Update to `kyverno` (app) version 0.14.4, restricting the namespaces where `PolicyExceptions` may be created.
```
